janusgraph-solr/pom.xml

```diff
@@ -56,19 +54,27 @@
             <groupId>org.apache.solr</groupId>
             <artifactId>solr-solrj</artifactId>
             <version>${solr.version}</version>
+            <exclusions>
+                <!-- Use jcl-over-slf4j from gremlin-core -->
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>jcl-over-slf4j</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.solr</groupId>
```

Comment:
@wojciechwojcik There's a direct dependency on solr-solrj (see `mvn dependency:analyze` results) and I think it's typically recommended to define such dependencies explicitly.